apps/ideaspace/src/components/common/IdeaCard/IdeaCardTag.js

```diff
@@ -1,31 +1,38 @@
 import React, { useState } from 'react';
-import theme from '@devlaunchers/components/styles/theme';
 import { StatuBox } from './StyledIdeaCard';
+import { orange } from '@mui/material/colors';
 
 const IdeaCardTag = ({ status }) => {
   const [tag, setTag] = useState('');
   const [tagColor, setTagColor] = useState('');
 
   React.useEffect(() => {
-    if (status == 'submitted') {
+    // const rootStyles = getComputedStyle(document.documentElement);
+    // const primaryColorValue = rootStyles.getPropertyValue('--orange-orange-100').trim();
+    // const lightBlueColorValue = rootStyles.getPropertyValue('--light-blue-lightblue-100').trim();
+    // const yellowColorValue = rootStyles.getPropertyValue('--yellow-yellow-100').trim();
+    // const successColorValue = rootStyles.getPropertyValue('--success-success-100').trim();
+    // const blueColorValue = rootStyles.getPropertyValue('--blue-blue-100').trim();
+
+    if (status === 'submitted') {
       setTag('submitted');
-      setTagColor(theme.colors.LIGHT_BLUE_100);
-    } else if (status == 'applying') {
+      setTagColor('--light-blue-lightblue-100');
+    } else if (status === 'applying') {
       setTag('in-review');
-      setTagColor(theme.colors.YELLOW_100);
-    } else if (status == 'approved') {
+      setTagColor('--yellow-yellow-100');
+    } else if (status === 'approved') {
       setTag('approved');
-      setTagColor(theme.colors.SUCCESS_100);
-    } else if (status == 'archived') {
+      setTagColor('--success-success-100');
+    } else if (status === 'archived') {
       setTag('archived');
-      setTagColor(theme.colors.BLUE_100);
+      setTagColor('--blue-blue-100');
     } else {
       setTag('workshopping');
-      setTagColor(theme.colors.ORANGE_100);
+      setTagColor('--orange-orange-100');
```

Comment:
a more maintainable and readable way to handle this is by using a key value pair of [status]: 'tailwind-class'

```
const tagColor = {
applying: 'yellow-yellow-100'
.
.
.
}


return <myDiv className={tagColor} />
```
then you won't need most of this logic and useEffect

Reply:
Hello @Enjoy2Live, I tried this approach, and I can't seem to access Tailwind CSS colors.  Perhaps there is something missing, for not being able to apply Tailwind CSS classes correctly.

Reply:
Hello @Hindu-Priya , you said you tried this approach but I'm not seeing the commit for your try.

can you please make a commit with my suggestion and push it so I can take a look?

Reply:
Hello @Enjoy2Live, I just committed my attempt based on your suggestion. Please take a look when you get a chance.